bciers/apps/registration/app/data/jsonSchema/operationRegistration/optedInOperationText.tsx

```diff
@@ -0,0 +1,117 @@
+const optedInOperationPreface = (
+  <div>
+    Complete the following fields to apply to be an Opt-in Operation in the B.C.
+    OBPS. Ensure that you have read through{" "}
+    <a
+      href="https://www2.gov.bc.ca/gov/content/environment/climate-change/industry/bc-output-based-pricing-system"
```

Comment:
Can we put all URLs in the constants file located in `bciers/libs/utils/urls.ts`?

Reply:
Good call, I wasn't aware of this file.